wca-regulations.md

```diff
@@ -413,11 +413,11 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
         - A7a1) If the judge finds that the puzzle is solved, they call "OKAY".
         - A7a2) If the judge assigns any penalties, they call "PENALTY".
         - A7a3) If the result is DNF, the judge calls "DNF".
-    - A7b) The judge records the result on a score sheet.
+    - A7b) The judge records the result on a score sheet and signs next to the result.
```

Comment:
Nope -- we're using cubingusa scorecards for CubingUSA Nationals, with group numbers printed on the scorecard, since we want to have everything tightly integrated with our scheduling and group-assigning software.

(of course, this is now the only competition left using cubingusa scorecards since all US competitions are migrated to the WCA site)

+1 to not changing things immediately before cubingusa nationals and worlds, either regulations or scorecard layouts.